tasks/tasks_prometheus_pgbouncer_exporter.yml

```diff
@@ -0,0 +1,58 @@
+---
+  - name: Set exporter facts
+    set_fact:
+      pgbouncer_exporter_sub_dir: "{{ exporter_config.exporter_dir | default('pgbouncer_exporter') }}"
+      pgbouncer_exporter_user: "{{ exporter_config.exporter_user | default(prometheus_user) }}"
+      pgbouncer_exporter_group: "{{ exporter_config.exporter_group | default(prometheus_group) }}"
+
+  - name: Prometheus pgbouncer exporter | ensure that python3 and dependencies are installed
+    become: yes
+    apt:
```

Comment:
As you noted, test matrix covers debian, centos, fedora 

More over (in addition to apt module) I would prefer customer to have a control over python3 installation.
Although you can assert for presents and fail early